examples/handling_files.q

```diff
@@ -11,13 +11,13 @@
 \d .protobufkdb
```

Comment:
Let's remove this line and append .protobufkdb. to function names. As an example functions having its full namespace is user friendly.

Reply:
done